### PR TITLE
Fix partner logo component

### PR DESCRIPTION
**Pull request by nerdlet**
## Description
Fixes #164296153

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Screenshots
image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/components/PartnerLogos.js [hunk 1]
```diff
@@ -69,29 +81,47 @@ const styles = theme => ({
...
+    <Grid container xs={12}>
+      <Grid
+        container
+        xs={12}
+        direction="row"
+        justify="center"
+        alignItems="center"
+        className={classes.title}
+      >
+        <Typography variant="h5">OUT PARTNERS</Typography>
```
Comment (kilemensi) [inline src/components/PartnerLogos.js]: **OUR** instead of **OUT**

Comment (kilemensi) [reply src/components/PartnerLogos.js]: If we're going with centred heading, then lets make it the same size as other headings such as Showcase, The Issues, and Sensors Stories.

Alternative design is Dominion country partners:

screenshot_2019-03-05 zeplin - project

Comment (nerdlet) [reply src/components/PartnerLogos.js]: @kilemensi The dominion design is what I originally had before the feedback changes. I'll change the heading instead to match the overall design of sensors.AFRICA

Comment (nerdlet) [reply src/components/PartnerLogos.js]: @kilemensi Fixed

**Pull request merged**